### PR TITLE
[Auditv2] Default copy save location

### LINKED ISSUE #34086: FE - Add smart defaults when duplicating content in audit collections
When clicking on `Make a copy` or `Duplicate` any content in the `Instance analytics` collection, the destination collection default should be `Custom reports`.

### LINKED ISSUE #35225: [AuditV2] Add basic e2e tests that ensure that audit content loads

### PR DESCRIPTION
**Pull request by iethree**
Closes https://github.com/metabase/metabase/issues/34086
Also includes https://github.com/metabase/metabase/pull/35342 and https://github.com/metabase/metabase/pull/35230

### Description

Defaults to saving copied instance analytics questions, models, and dashboards to the "Custom Reports" sub-collection.

Screen Shot 2023-10-27 at 11 23 52 AM

### Checklist

- [x] Tests have been added/updated to cover changes in this PR

### CODE REVIEW

File: e2e/test/scenarios/collections/instance-analytics.cy.spec.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        cy.findByTextEnsureVisible("Custom reports");
+        cy.button("Duplicate").click();
+      });
+
+      cy.wait("@copyDashboard").then(({ response }) => {
+        expect(response.statusCode).to.eq(200);
+      });
+    });
+
+    it.skip("should not allow moving or archiving custom reports collection", () => {
```
Comment (iethree) [inline e2e/test/scenarios/collections/instance-analytics.cy.spec.js:120]: this is a test for https://github.com/metabase/metabase/pull/35230, will unskip it when that merges

**Review by npfitz — CHANGES_REQUESTED** [review]
Found a small issue where we show an error if we save a question, then try to duplicate it immediately. This doesn't happen if you save the question, go back home, then re-open the question, so I'm not sure what's going on here.

chrome_6O9ddjgqOQ

**Comment by iethree** [comment]
> Found a small issue where we show an error if we save a question, then try to duplicate it immediately. This doesn't happen if you save the question, go back home, then re-open the question, so I'm not sure what's going on here.

Collections loading state issue. Good catch! 👍

**Review by npfitz — APPROVED** [review]
Looks Good 🔥

**Pull request merged**